drlhp/deprecated/run.py

```diff
@@ -9,17 +9,14 @@
 
 import cloudpickle
 import easy_tf_log
-from a2c import logger
-from a2c.a2c.a2c import learn
-from a2c.a2c.policies import CnnPolicy, MlpPolicy
-from a2c.common import set_global_seeds
-from a2c.common.vec_env.subproc_vec_env import SubprocVecEnv
-from params import parse_args, PREFS_VAL_FRACTION
-from pref_db import PrefDB, PrefBuffer
-from pref_interface import PrefInterface
-from reward_predictor import RewardPredictorEnsemble
-from reward_predictor_core_network import net_cnn, net_moving_dot_features
-from utils import VideoRenderer, get_port_range, make_env
+from drlhp.deprecated.a2c import logger, learn
```

Comment:
Ignoring this file since it seems the normal way to use this code is through the env wrapper, and the A2C part of this code looked like it was in the middle of being debugged.